tests/Description/AbstractPriceDescriptionTest.php

```diff
@@ -1,7 +1,10 @@
 <?php
+namespace Blesta\Pricing\Tests\Unit\Description;
```

Comment:
Namespace should be `Blesta\Pricing\Tests\Description;` *OR* test should be moved into a `Unit` directory within `tests/` (notice case).